apps/api/src/app/events/usecases/parse-event-request/parse-event-request.command.ts

```diff
@@ -39,6 +48,10 @@ export class ParseEventRequestBaseCommand extends EnvironmentWithUserCommand {
   @IsEnum(TriggerRequestCategoryEnum)
   requestCategory?: TriggerRequestCategoryEnum;
 
+  @IsOptional()
+  @IsNumber()
+  priority?: number;
```

Comment:
Please apply the same conditional for `ApiHideProperty` here [as discussed in the 0.24x PR](https://github.com/novuhq/novu/pull/6528#discussion_r1768455318).

Reply:
Done